src/MetaMorpho.sol

```diff
@@ -712,10 +747,8 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
             uint256 toSupply = UtilsLib.min(_suppliable(marketParams, id), assets);
 
             if (toSupply > 0) {
-                // Using try/catch to skip markets that revert.
-                try MORPHO.supply(marketParams, toSupply, 0, address(this), hex"") {
-                    assets -= toSupply;
-                } catch {}
+                MORPHO.supply(marketParams, toSupply, 0, address(this), hex"");
+                assets -= toSupply;
```

Comment:
So we assume that allocators allocators remove broken markets from the supply queue ? (I'm fine with it btw)

Reply:
Yes I think this is a good assumption